sound/soc/sof/ipc4-topology.c

```diff
@@ -893,18 +890,6 @@ static int sof_ipc4_widget_setup_comp_process(struct snd_sof_widget *swidget)
 	if (process->payload_with_output_fmt)
```

Comment:
@ujfalusi while you are at it. Can you please also fix this part? Looks like init instance with output format isnt supported either right? The wov module needs this and will just have to adapt to use the base_cfg_ext right?